ImprimirAutor.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+require_once 'Autor.php';
```

Comment:
```suggestion
require_once "Autor.php";
```
Usar un solo estilo de comillas, ya sean simples o dobles